tools/flaky-test-reporter/result.go

```diff
@@ -113,6 +113,19 @@ func getFlakyRate(rd *RepoData) float32 {
 	return float32(len(getFlakyTests(rd))) / float32(totalCount)
 }
 
+func flakyRateAboveThreshold(rd *RepoData) bool {
+	// if the percent determined by the test count threshold is higher than
+	// the percent threshold, use that instead of the percent threshold
+	var threshold float32
```

Comment:
you dont need the else.
```
	threshold := float32(countThreshold) / float32(totalCount)
```

Reply:
the main reason for doing this was to ensure totalCount wasn't zero, though that _shouldn't_ happen (no test jobs should have zero tests). I do agree this would be cleaner.

Reply:
we can have a 
```
// Num of tests = 0 => no tests are flaky :)
if (totalCount == 0) {
return true
}

threshold := float32(countThreshold) / float32(totalCount)
if ...